tests/cases/compiler/jsFileFunctionParametersAsOptional2.ts

```diff
@@ -0,0 +1,18 @@
+// @allowJs: true
+// @noEmit: true
+
+// @filename: foo.js
+/**
+ * @param a
+ * @param b
+ * @param c

```

Comment:
you might have a test case that shows that additional, undocumented parameters are still not optional. This is probably not *intended* but it is the way the code will behave.